Cargo.toml

```diff
@@ -12,6 +12,7 @@ edition = "2018"
 [dependencies]
 chrono = "0.4.6"
 xml-rs = "0.8.0"
+derive-getters = "0.0.7"
```

Comment:
It's not clear to me why this is needed? If this is for some specific use case it might be better to put this behind a feature flag.

Reply:
This is just to keep fields private and give structures read-only way to access them.